articles/101_build_tool.md

```diff
@@ -3,7 +3,7 @@ layout: default
 title: A universal build tool
 permalink: articles/build_tool.html
 abstract:
-  This article describes the rationale for a universal build tool.
+  This article describes a universal build tool for ROS 1 and ROS 2.
```

Comment:
as part of the goals is to have this build tool ROS independent. Do we need to narrow it to ROS 1 and ROS 2 here?